debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java

```diff
@@ -94,6 +94,8 @@ public ChangeEventSourceCoordinator<PostgresPartition, PostgresOffsetContext> st
         final Charset databaseCharset;
         try (PostgresConnection tempConnection = new PostgresConnection(connectorConfig.getJdbcConfig(), PostgresConnection.CONNECTION_GENERAL)) {
             databaseCharset = tempConnection.getDatabaseCharset();
+        } catch (Exception e)  {
+            throw new RetriableException("Couldn't obtain encoding for database", e);
```

Comment:
```suggestion
        } 
        catch (Exception e)  {
            throw new RetriableException("Couldn't obtain encoding for database", e);
```
Just to make the formatter happy.